### PR TITLE
Governance docs edits

### PR DESCRIPTION
**Pull request by 0xElihu**
### Attention (CC)

@LongForWisdom, @blimpa 

### Summary of Changes

I rewrote parts of the two docs for better clarity and fixed some typos.

### CODE REVIEW

File: content/en/learn/governance/module-psm.md [hunk 1]
```diff
@@ -17,29 +17,29 @@ Scope: Each PSM interacts with a single underlying vault type.
...
 
 Additionally, the Maker Protocol bears the risk of the underlying collateral stablecoin losing its peg (though this is no different from regular vault-types with stablecoin collateral.)
 
 On the other hand, a PSM offers several advantages:
 
 - Increased DAI stability due to the instant arbitrage opportunity when the DAI price diverges from the price of the collateral type underlying the PSM.
-- Fees are taken upfront on each swap to and from DAI. Because a PSM has no slippage, it is expected that it will attract respectable trading volume if given reasonable fee parameters.
-- There is no requirement to micromanage parameters to ensure continued functioning like there is with low liquidation ratio stablecoin vaults.
+- Fees are taken upfront on each swap to and from DAI. Because a PSM has no slippage, it's able to attract respectable trading volume.
+- There is no requirement to micromanage parameters (via governance actions) to ensure continued functioning, when compared to low liquidation ratio stablecoin vaults.
```
Comment (blimpa) [inline content/en/learn/governance/module-psm.md:42]: "in contrast to [low liquidation...]" would probably work better here, I think

File: content/en/learn/governance/module-dciam.md [hunk 2]
```diff
@@ -32,11 +32,11 @@ The module was designed with two purposes in mind:
 
 The DC-IAM provides a number of benefits and only one known downside. The main benefits are the reduced governance overhead in setting debt ceilings and risk mitigation in the event of a large decrease in collateral price. These benefits are significant.
 
-The one minor downside is that the DC-IAM enables a griefing attack on the Debt Ceiling of collateral types that are using it. The outcome of this attack is to prevent DAI from being minted using a given collateral type. The severity and likelihood of this attack have been determined to be minimal and the mitigation strategy is straightforward (disable the DC-IAM for the attacked collateral type.)
+The one minor downside is that the DC-IAM enables a griefing attack on the Debt Ceiling of collateral types that are using it. The outcome of this attack is to prevent DAI from being minted using a given collateral type. The severity and likelihood of this attack have been determined to be minimal and the mitigation strategy is straightforward, i.e., disable the DC-IAM for the attacked collateral type.
```
Comment (blimpa) [inline content/en/learn/governance/module-dciam.md:35]: Maybe "namely, disabling..."?

File: content/en/learn/governance/module-psm.md [hunk 3]
```diff
@@ -17,29 +17,29 @@ Scope: Each PSM interacts with a single underlying vault type.
 
 A Peg Stability Module allows users to swap a given collateral type directly for DAI at a fixed rate, rather than borrowing DAI. The PSM contract was designed with stablecoin collateral in mind, allowing users to swap other stablecoins for DAI at a fixed rate to aid with keeping DAI pegged to 1 dollar.
 
-A PSM operates similarly to a regular vault type with a zero stability fee and a liquidation ratio of 100% that can only be accessed through a user-facing smart contract containing the relevant swap functions. Unlike a regular vault, a user of a PSM does not retain ownership of the asset and borrow DAI, instead they swap the asset directly for DAI.
+A PSM operates similarly to a regular vault type with a zero stability fee and a liquidation ratio of 100% that can only be accessed through a user-facing smart contract containing the relevant swap functions. Unlike a regular vault, PSM's user doesn't retain ownership of the asset and borrow DAI, instead they swap the asset directly for DAI.
```
Comment (blimpa) [inline content/en/learn/governance/module-psm.md:20]: "Unlike the case with regular vaults, users of PSM don't retain..."?

File: content/en/learn/governance/module-psm.md (hunk 1 above)
Comment (0xElihu) [reply content/en/learn/governance/module-psm.md:42]: Changed!

File: content/en/learn/governance/module-dciam.md (hunk 2 above)
Comment (0xElihu) [reply content/en/learn/governance/module-dciam.md:35]: done

File: content/en/learn/governance/module-psm.md (hunk 3 above)
Comment (0xElihu) [reply content/en/learn/governance/module-psm.md]: changed, thanks!

**Comment by LongForWisdom** [comment]
@blimpa how is this looking now?

**Comment by blimpa** [comment]
> @blimpa how is this looking now?

Looks good to me

**Comment by 0xElihu** [comment]
@LongForWisdom, please merge.

**Pull request merged**